lsp-mode.el

```diff
@@ -4814,6 +4820,9 @@ Return `nil' when fails to guess prefix."
                            'lsp-completion-score score?))
              it)))
 
+(defvar lsp--capf-no-reordering nil
```

Comment:
seems like this should be public?

Reply:
I can make this public, but can I keep this as `defvar`, since this option is dangerous (and quirky).
Alternatively, the option should be set is `lsp-completion-no-cache`.